main.js

```diff
@@ -545,3 +554,25 @@ function getCurrency (currencyString, currencySign) {
 	  }
 	});
 }
+function dirtyWords (wordToDefine) {
```

Comment:
for the love of god indent your code properly

Reply:
this is really for the if and else parts

Reply:
I do indent it properly, it gets changed when i push it somehow?